api_webhooks.go

```diff
@@ -131,7 +131,7 @@ func (client *Client) GetQueuedTaskUpdates(webhookId string) (response dtos.Sing
 	}
 
 	response = make(dtos.SingularityTaskHistoryUpdateList, 0)
-	err = client.DTORequest(&response, "GET", "/api/webhooks/task", pathParamMap, queryParamMap)
+	err = client.DTORequest("singularity-getqueuedtaskupdates", &response, "GET", "/api/webhooks/task", pathParamMap, queryParamMap)
```

Comment:
Are these strings all written by hand or are they generated somehow?

Reply:
I saw the generation code over in https://github.com/opentable/swaggering/pull/4#discussion_r155491838 neat :)